js/packages/cli/src/commands/verifyTokenMetadata/index.ts

```diff
@@ -90,7 +90,9 @@ export const verifyCreatorCollation = (
 };
 
 export const verifyImageURL = (image, files, manifestFile) => {
-  const expectedImagePath = `image${EXTENSION_PNG}`;
+  // The image is expected to have the same name as the index
+  const fileIndex = manifestFile.split('/').pop().split('.')[0];
```

Comment:
Wondering if using [url](https://nodejs.org/api/url.html) or similar would do a better job at pulling out parts of this url.